database/StudentServiceInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ilija.savic
+ * Date: 8/24/2017
+ * Time: 9:42 AM
+ */
+
+namespace database;
+
+interface StudentServiceInterface
```

Comment:
is this interface only valid for Student model or? name of interface is kinda misleading